appium/webdriver/connectiontype.py

```diff
@@ -26,7 +26,7 @@
 """
 
 
-class ConnectionType(object):
+class ConnectionType:
```

Comment:
[Notes]
Couldn't use `enum` here since `ConnectionType(Enum)` will be different type from below return value.

https://github.com/appium/python-client/blob/e16ab705a218b302dec9c61560af44c835051692/appium/webdriver/extensions/android/network.py#L69

Need to change such like `return CoonectionType(self.execute(Command.SET_NETWORK_CONNECTION, data)['value'])`, but this breaks backward compatibility.